dvc/params.yaml

```diff
@@ -76,50 +35,6 @@ models-and-features:
     model: xgb
     features-rd: image-tmd-rd
     features-proj: image-tmd-proj
-feature_extractors:
-  in_deepwalk:
-    dataset_name: interneurons
-    feature_extractor_name: deepwalk
-    id: in_deepwalk
-    input_csv: data/prepared/interneurons/dataset_features_labels.csv
-    output_wildcard: '''data/prepared/interneurons/*/*/*_deepwalk.pickle'''
-  in_morph:
-    dataset_name: interneurons
-    feature_extractor_name: morphology
-    id: in_morph
-    input_csv: data/prepared/interneurons/dataset_features_labels.csv
-    output_wildcard: ''
-  in_tmd:
-    dataset_name: interneurons
-    feature_extractor_name: tmd
-    id: in_tmd
-    input_csv: data/prepared/interneurons/dataset_features_labels.csv
-    output_wildcard: '''data/prepared/interneurons/*/*/*_tmd.pickle'''
-  pc_deepwalk:
-    dataset_name: pyramidal_cells
-    feature_extractor_name: deepwalk
-    id: pc_deepwalk
-    input_csv: data/prepared/pyramidal_cells/dataset_features_labels.csv
-    output_wildcard: '''data/prepared/pyramidal_cells/*/*/*_deepwalk.pickle'''
-  pc_morph:
-    dataset_name: pyramidal_cells
-    feature_extractor_name: morphology
-    id: pc_morph
-    input_csv: data/prepared/pyramidal_cells/dataset_features_labels.csv
-    output_wildcard: ''
-  pc_tmd:
-    dataset_name: pyramidal_cells
-    feature_extractor_name: tmd
-    id: pc_tmd
-    input_csv: data/prepared/pyramidal_cells/dataset_features_labels.csv
-    output_wildcard: '''data/prepared/pyramidal_cells/*/*/*_tmd.pickle'''
-performance-report:
```

Comment:
Not used any more, because the DVC stage that used this (`performance-report`) item was removed.